infra/cleanup.sh

```diff
@@ -27,6 +27,10 @@ echo "IDENTITY_LIST: $IDENTITY_LIST"
 AUTOSCALE_SETTINGS_LIST=$(az monitor autoscale list  --query "[*].[name]" -o tsv | grep -E "autoscale-" | sort -u)
 echo "AUTOSCALE_SETTINGS_LIST: $AUTOSCALE_SETTINGS_LIST"
 
+#Query Workspaces
+WORKSPACE_LIST=$(az ml workspace list --query "[*].[name]" -o tsv)
```

Comment:
I would recommend adding a filter on the query for workspaces. Otherwise the print statement will show all the workflows.

$(az ml workspace list --query "[*].[name]" -o tsv | grep -E "mlw-basic-prod-|mlw-basicex-prod-" | sort -u)

Reply:
Makes sense. Changed accordingly.